site/content/extend/plugins/webapp/proper-configuration.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Proper Configuration
+subsection: Web App Plugins
+weight: 0
+---
+
+### Developing in the plugin's webapp folder
```

Comment:
When we refer to the repo, we say "webapp" but when we refer to anything other than the repo (i.e., the web app proper) I propose we refer to "web app". 

In this case are we referring to the repo or the name of a folder that follows the structure of the repo and is called "webapp"?

Reply:
> In this case are we referring to the repo or the name of a folder that follows the structure of the repo and is called "webapp"?

There is a folder in each plugin repo called `webapp`, which houses each plugin's code that is modifying the Mattermost web application's user interface.

Reply:
Gotcha, that makes sense. :)

In that case we can use ```webapp``` in the header.